### PR TITLE
add DLRG qualifications

### PR DESCRIPTION
**Pull request by jeriox**
adds DLRG qualifications as stated in #78

### CODE REVIEW

File: ephios/extra/management/commands/setupdata.py [hunk 1]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+        )
+        tauchf.included_qualifications.add(taucher_2)
+
+        boats_category = QualificationCategory.objects.create(
+            title=_("Boats (DLRG)"),
+            uuid=uuid.UUID("f1e2619b-56ee-4e4a-b429-b923f8586771"),
+        )
+        Qualification.objects.create(
+            category=boats_category,
+            title="Bootsführerschein Binnen (DLRG)",
```
Comment (Steditor) [inline ephios/extra/management/commands/setupdata.py:397]: Formally, this is called "DLRG-Bootsführerschein A"

File: ephios/extra/management/commands/setupdata.py [hunk 2]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+            abbreviation="Bootsführerschein Binnen",
+            uuid=uuid.UUID("7ec0c9e2-f211-4370-b796-e309f3e9b448"),
+        )
+        Qualification.objects.create(
+            category=boats_category,
+            title="Bootsführerschein See (DLRG)",
```
Comment (Steditor) [inline ephios/extra/management/commands/setupdata.py:403]: Formally, this is called "DLRG-Bootsführerschein B"

File: ephios/extra/management/commands/setupdata.py [hunk 3]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+            uuid=uuid.UUID("1ba25ee1-fa88-4d44-9416-3a024caa6daf"),
+        )
+
+        wr_category = QualificationCategory.objects.create(
+            title=_("Water rescue (DLRG)"),
+            uuid=uuid.UUID("93574a61-1a6e-4dd5-8df3-40e8c51b693f"),
+        )
+        rettschw = Qualification.objects.create(
+            category=wr_category,
+            title="Rettungsschwimmer",
```
Comment (Steditor) [inline ephios/extra/management/commands/setupdata.py:414]: This should be qualified as "Rettungsschwimmer Silber".

File: ephios/extra/management/commands/setupdata.py [hunk 4]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+        )
+        sr2.included_qualifications.add(sr1)
+
+        radio_category = QualificationCategory.objects.create(
+            title=_("Radio"),
+            uuid=uuid.UUID("368fb612-f8c5-476a-a4e7-1478996a8ab3"),
+        )
+        Qualification.objects.create(
+            category=radio_category,
+            title="Digitalfunk",
```
Comment (Steditor) [inline ephios/extra/management/commands/setupdata.py:446]: Formally this is called "BOS-Sprechfunker -digital-". Maybe one wants to include the reference to "BOS"?

All other entries are named according to a qualification, this one should be, too.

File: ephios/extra/management/commands/setupdata.py [hunk 5]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+            abbreviation="Digitalfunk",
+            uuid=uuid.UUID("506c4d4c-df11-4d41-95d7-c4ddfc78b706"),
+        )
+        Qualification.objects.create(
+            category=radio_category,
+            title="Analogfunk",
+            abbreviation="Analogfunk",
```
Comment (Steditor) [inline ephios/extra/management/commands/setupdata.py:453]: Formally this is called "BOS-Sprechfunker -analog-". Maybe one wants to include the reference to "BOS"?

Also: All other entries are named according to a qualification, this one should be, too.

File: ephios/extra/management/commands/setupdata.py [hunk 6]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+            uuid=uuid.UUID("4974ea66-2086-4987-958b-503de21a285a"),
+        )
+        Qualification.objects.create(
+            category=radio_category,
+            title="Betriebsfunk (DLRG)",
```
Comment (Steditor) [inline ephios/extra/management/commands/setupdata.py:458]: The qualification is called "DLRG-Sprechfunker". All other entries are named according to a qualification, this one should be, too.

File: ephios/extra/management/commands/setupdata.py [hunk 7]
```diff
@@ -145,13 +145,30 @@ def create_objects(self, *args, **options):
...
+        sanb.included_qualifications.add(sana)
+
         sanh = Qualification.objects.create(
             category=medical_category,
             title="Sanitätshelfer",
             abbreviation="SanH",
             uuid=uuid.UUID("b1faab38-2e7c-4507-b753-06d1e653412d"),
         )
         sanh.included_qualifications.add(eh)
+        sanh.included_qualifications.add(sanb)
```
[the remark this answers is not in the document]
Comment (felixrindt) [reply ephios/extra/management/commands/setupdata.py:171]: :man_facepalming:

File: ephios/extra/management/commands/setupdata.py (hunk 3 above)
Comment (SimonLickteig) [reply ephios/extra/management/commands/setupdata.py:414]: I think it doesn't matter if it is "Rettungsschwimmer Silber" or "Rettungsschwimmer gold". So "Rettungsschwimmer" ist a general designation.

File: ephios/extra/management/commands/setupdata.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (SimonLickteig) [reply ephios/extra/management/commands/setupdata.py:171]: Jep! That is very stupid at the dlrg. Die official translation of "SanA" ist "Sanitätshelfer". But it is possible that a person has a real "SanH" (like the qualification from another organisation like Johanniter, Malteser, ASB etc.). The workload from the SanH ist min. 48h. SanA ist only 24h. 
I think you can compare the SanH with SanB. 

"SanA" and "SanB" are specific DLRG qualifications. SanH, EH, NFS, RetSan, RH, RettAss, NA are general qualifications.

But I don't know what to do for ephios 😆

File: ephios/extra/management/commands/setupdata.py [hunk 8]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+            title="Einsatztaucher Stufe 2",
+            abbreviation="Einsatztaucher Stufe 2",
+            uuid=uuid.UUID("481fc9fb-e90a-4b83-8a07-456034aa0a33"),
+        )
+        taucher_2.included_qualifications.add(taucher_1)
+
+        tauchf = Qualification.objects.create(
+            category=diving_category,
+            title="Taucheinsatzführer",
+            abbreviation="Taucheinsatzführer",
```
Comment (SimonLickteig) [inline ephios/extra/management/commands/setupdata.py:386]: abbreviation = TEF

File: ephios/extra/management/commands/setupdata.py [hunk 9]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+            title="Einsatztaucher Stufe 1",
+            abbreviation="Einsatztaucher Stufe 1",
+            uuid=uuid.UUID("52fc6a88-6e56-4af5-8021-b8a6412f405c"),
+        )
+        taucher_1.included_qualifications.add(signal)
+
+        taucher_2 = Qualification.objects.create(
+            category=diving_category,
+            title="Einsatztaucher Stufe 2",
+            abbreviation="Einsatztaucher Stufe 2",
```
Comment (SimonLickteig) [inline ephios/extra/management/commands/setupdata.py:378]: abbreviation = ET2

File: ephios/extra/management/commands/setupdata.py [hunk 10]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+            category=diving_category,
+            title="Signalmann",
+            abbreviation="Signalmann",
+            uuid=uuid.UUID("436aaab0-5e8f-43aa-8f33-7d9ea68f7973"),
+        )
+
+        taucher_1 = Qualification.objects.create(
+            category=diving_category,
+            title="Einsatztaucher Stufe 1",
+            abbreviation="Einsatztaucher Stufe 1",
```
Comment (SimonLickteig) [inline ephios/extra/management/commands/setupdata.py:370]: abbreviation = ET1

File: ephios/extra/management/commands/setupdata.py [hunk 11]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+        Qualification.objects.create(
+            category=boats_category,
+            title="Bootsführerschein Binnen (DLRG)",
+            abbreviation="Bootsführerschein Binnen",
+            uuid=uuid.UUID("7ec0c9e2-f211-4370-b796-e309f3e9b448"),
+        )
+        Qualification.objects.create(
+            category=boats_category,
+            title="Bootsführerschein See (DLRG)",
+            abbreviation="Bootsführerschein See",
```
Comment (SimonLickteig) [inline ephios/extra/management/commands/setupdata.py:404]: abbreviation = BF B

File: ephios/extra/management/commands/setupdata.py [hunk 12]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+        tauchf.included_qualifications.add(taucher_2)
+
+        boats_category = QualificationCategory.objects.create(
+            title=_("Boats (DLRG)"),
+            uuid=uuid.UUID("f1e2619b-56ee-4e4a-b429-b923f8586771"),
+        )
+        Qualification.objects.create(
+            category=boats_category,
+            title="Bootsführerschein Binnen (DLRG)",
+            abbreviation="Bootsführerschein Binnen",
```
Comment (SimonLickteig) [inline ephios/extra/management/commands/setupdata.py:398]: abbreviation = BF A

File: ephios/extra/management/commands/setupdata.py [hunk 13]
```diff
@@ -328,6 +345,122 @@ def create_objects(self, *args, **options):
...
+        rettschw = Qualification.objects.create(
+            category=wr_category,
+            title="Rettungsschwimmer",
+            abbreviation="Rettungsschwimmer",
+            uuid=uuid.UUID("ef95a854-2eeb-431c-a795-bc291b341d49"),
+        )
+        wr = Qualification.objects.create(
+            category=wr_category,
+            title="Fachausbildung Wasserrettungsdienst (DLRG)",
+            abbreviation="Wasserretter",
```
Comment (SimonLickteig) [inline ephios/extra/management/commands/setupdata.py:421]: abbreviation = WR

File: ephios/extra/management/commands/setupdata.py (hunk 3 above)
Comment (Steditor) [reply ephios/extra/management/commands/setupdata.py:414]: Well - there is also "Bronze", and we usually don't consider that to be enough qualification for water rescue missions.

File: ephios/extra/management/commands/setupdata.py (hunk 7 above)
[the remark this answers is not in the document]
Comment (jeriox) [reply ephios/extra/management/commands/setupdata.py:171]: I discussed that with @SimonLickteig and we decided to keep the Sanitätshelfer as the qualification that is known from the other Hios and in fact includes the San B. This means that the DLRG users that also use the term Sanitätshelfer need to watch out and select the correct qualification, but I think that's kind of acceptable and we didn't come up with a better solution

File: ephios/extra/management/commands/setupdata.py (hunk 3 above)
Comment (jeriox) [reply ephios/extra/management/commands/setupdata.py:414]: we decided to include them all

**Review by SimonLickteig — COMMENTED** [review]
Yep

File: ephios/extra/management/commands/setupdata.py [hunk 14]
```diff
@@ -328,6 +345,135 @@ def create_objects(self, *args, **options):
...
+            abbreviation="RS Silber",
+            uuid=uuid.UUID("ef95a854-2eeb-431c-a795-bc291b341d49"),
+        )
+        rettschw_silber.included_qualifications.add(rettschw_bronze)
+        rettschw_gold = Qualification.objects.create(
+            category=wr_category,
+            title="Rettungsschwimmer Gold",
+            abbreviation="RS Gold",
+            uuid=uuid.UUID("b601a18b-cee8-4037-af33-dd7aabeac295"),
+        )
```
Comment (felixrindt) [inline ephios/extra/management/commands/setupdata.py:430]: Rettungsschwimmen is not specific to DLRG. we could do a seperate DRSA-Catergory

**Review by felixrindt — APPROVED** [review]

**Pull request merged**